prow/tide/tide.go

```diff
@@ -49,8 +49,12 @@ import (
 	"k8s.io/test-infra/prow/tide/history"
 )
 
-// For mocking out sleep during unit tests.
-var sleep = time.Sleep
+var (
+	// For mocking out sleep during unit tests.
+	sleep = time.Sleep
+
+	mergeFailureComment = "Attempt to merge the PR failed. Tide may retry depending on the type of failure."
```

Comment:
Is it not possible to know whether or not we will retry? I worry that this will cause a larger support load ("I'm a user, and I'm seeing this vague comment, what does it mean?")

Reply:
Thank you for the review. Fixed.